pkg/deploy/kubernetes_image_puller.go

```diff
@@ -449,15 +449,11 @@ func GetDefaultImages() []ImageAndName {
 		"^RELATED_IMAGE_.*_plugin_openshift$",
```

Comment:
I suspect all these _plugin_ values could be collapsed into `.*_devfile_registry_image.*` or `.*_plugin_registry_image.*`

Reply:
See content in https://github.com/redhat-developer/codeready-workspaces-images/blob/crw-2-rhel-8/codeready-workspaces-operator-metadata-generated/manifests/codeready-workspaces.csv.yaml

Reply:
Thank you for the review, I realized that the `^RELATED_IMAGE_.*_plugin_java8` can be collapsed into `^RELATED_IMAGE_.*_java.{1,2}(_maven)?_devfile_registry_image.*`, since in [1], the [`java8_plugin_registry_image`](https://github.com/redhat-developer/codeready-workspaces-images/blob/0d99d4695a325422bbb0f7bf65558735b9ce40f3/codeready-workspaces-operator-metadata-generated/manifests/codeready-workspaces.csv.yaml#L981) and [`java8_devfile_registry_image`](https://github.com/redhat-developer/codeready-workspaces-images/blob/0d99d4695a325422bbb0f7bf65558735b9ce40f3/codeready-workspaces-operator-metadata-generated/manifests/codeready-workspaces.csv.yaml#L975) refer to the same image.

I've left `.*_devfile_registry_image.*` and `.*_plugin_registry_image.*` out for now, since they might be a bit too lenient for images in [2]

[1]  https://github.com/redhat-developer/codeready-workspaces-images/blob/crw-2-rhel-8/codeready-workspaces-operator-metadata-generated/manifests/codeready-workspaces.csv.yaml, 
[2] https://github.com/eclipse-che/che-operator/blob/main/bundle/stable/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml